### PR TITLE
fix: Restore the spec ovs_appctl_fdb_show_bridge

### PR DESCRIPTION
**Pull request by psachin**
### Description:

Restore the spec which was removed in https://github.com/RedHatInsights/insights-core/commit/eb70e566a327b12445dbba4b6ca05b90772effa0

Signed-off-by: Sachin Patil <psachin@redhat.com>

### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [X] Is this PR to correct an issue?
* [X] Is this PR an enhancement?

### Complete Description of Additions/Changes:

*Add your description here*

### CODE REVIEW

File: insights/specs/default.py [hunk 1]
```diff
@@ -484,6 +484,8 @@ class DefaultSpecs(Specs):
     ose_node_config = simple_file("/etc/origin/node/node-config.yaml")
     ovirt_engine_server_log = simple_file("/var/log/ovirt-engine/server.log")
     ovirt_engine_ui_log = simple_file("/var/log/ovirt-engine/ui.log")
+    ovs_vsctl_list_br = simple_command("/usr/bin/ovs-vsctl list-br")
```
Comment (bfahr) [inline insights/specs/default.py:487]: @psachin these also need to be added to the JSON files for collection.  The `ovs_vsctl_list_br` command needs to be added to insights/specs/insights_archive.py.

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/specs/default.py (hunk 1 above)
Comment (bfahr) [reply insights/specs/default.py:487]: Nevermind, now I see what you are doing.

File: insights/specs/insights_archive.py [hunk 2]
```diff
@@ -196,6 +196,7 @@ class InsightsArchiveSpecs(Specs):
     od_cpu_dma_latency = simple_file("insights_commands/od_-An_-t_d_.dev.cpu_dma_latency")
     open_vm_tools_stat_raw_text_session = simple_file("insights_commands/vmware-toolbox-cmd_stat_raw_text_session")
     openvswitch_other_config = simple_file("insights_commands/ovs-vsctl_-t_5_get_Open_vSwitch_._other_config")
+    ovs_appctl_fdb_show_bridge = glob_file("insights_commands/ovs-appctl_fdb.show_*")
```
Comment (bfahr) [inline insights/specs/insights_archive.py:199]: Foreach execute only works with core collection so this spec is not needed.

Comment (psachin) [reply insights/specs/insights_archive.py]: Got it.

**Review by xiangce — APPROVED** [review]

**Pull request merged**